docs/simulating_with_the_mvs.html

```diff
@@ -0,0 +1,678 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Why did you change this file to html? It will make it hard to edit it later on. Or did you compile this page seperately and it should not be uploaded at all?